googletest/CMakeLists.txt

```diff
@@ -321,3 +321,23 @@ if (gtest_build_tests)
   py_test(gtest_xml_output_unittest --no_stacktrace_support)
   py_test(googletest-json-output-unittest --no_stacktrace_support)
 endif()
+
+set_target_properties(
```

Comment:
Add same pdb support for targets gmock and gmock_main and copy all pdb to package